File: 2974-minimum-number-game/2974-minimum-number-game.cpp
class Solution {
public:
    vector<int> numberGame(vector<int>& nums) {
        sort(nums.begin(),nums.end());
        vector<int> ans;
        for(int i=0;i<nums.size();i+=2)
        {
            ans.push_back(nums[i+1]);
            ans.push_back(nums[i]);
        }
        return ans;
    }
};

File: 2974-minimum-number-game/2974-minimum-number-game.java
class Solution {
    public int[] numberGame(int[] arr) {
        Arrays.sort(arr);
        int n = arr.length;

        for(int i=0; i<n-1; i=i+2){
            int a = arr[i];
            arr[i] = arr[i+1];
            arr[i+1] = a;
        }

        return arr;

    }
}